azure-pipelines.yml

```
@@ -30,10 +30,12 @@ jobs:
       fi
 
       # Get the directories holding all changed files
-      PROJECTS=$(git diff-tree --no-commit-id --name-only -r HEAD "${COMPARE_AGAINST}" | grep -E ".+/.+" | cut -d/ -f1,2 | sort -u)
+      # 1. All changes
+      # 2. Only directories
+      # 3. max-depth 3
+      PROJECTS=$(git diff-tree --no-commit-id --name-only -r HEAD "${COMPARE_AGAINST}" | grep -E ".+/.+" | cut -d/ -f1,3 | sort -u)
```

```suggestion
      PROJECTS=$(git diff-tree --no-commit-id --name-only -r HEAD "${COMPARE_AGAINST}" | grep -E ".+/build_tarballs.jl" | sort -u)
```

Might be simpler, but then that also needs changes down the line